.github/workflows/loristest.yml

```diff
@@ -37,7 +38,7 @@ jobs:
           ${{ runner.os }}-php-
 
     - name: Change PHP Version in Dockerfile
-      run: sed -i "s/7.3/${{ matrix.php }}/g" Dockerfile.test.php7
+      run: sed -i "s/7.4/${{ matrix.php }}/g" Dockerfile.test.php7
```

Comment:
Why is this change needed? If the script is doing what it should then it shouldn't matter if it's "7.3", "7.4" or "apple".. this makes me worried that it's not properly testing 8.0.

Reply:
7.4 is the default version of the Docker PHP file.
if will be replaced by matrix.php (7.4 or 8.0) .
![Screen Shot 2021-02-05 at 10 58 09 AM](https://user-images.githubusercontent.com/9876007/107057434-737e8880-67a1-11eb-8cff-b2f57da392a7.png)
![Screen Shot 2021-02-05 at 10 57 44 AM](https://user-images.githubusercontent.com/9876007/107057454-7b3e2d00-67a1-11eb-96c9-b8dc1fba85a4.png)